.github/workflows/windows-workflow.yml

```diff
@@ -95,10 +94,10 @@ jobs:
 
       - name: Prepare Artifact Folder
         shell: bash
-        working-directory: ./bin
+        working-directory: bin
         run: |
           shopt -s extglob dotglob
-          mkdir "${{ steps.artifact-metadata.outputs.name }}"
+          mkdir -p "${{ steps.artifact-metadata.outputs.name }}"
```

Comment:
is there a case where the folder is recursive?

Reply:
No, it's just for safety.

Reply:
I don't have any strong feelings on that one, it's fine either way to me, so :shrug: